corehq/apps/events/models.py

```diff
@@ -137,28 +144,15 @@ def _unassign_attendees(self, attendees_case_ids):
         remove_indices_from_deleted_cases(self.domain, extension_cases_ids)
 
 
-class Attendee(models.Model):
-
-    ATTENDEE_CASE_TYPE = 'commcare-attendee'
-    EVENT_ATTENDEE_CASE_TYPE = 'commcare-potential-attendee'
-
-    domain = models.CharField(max_length=255)
-    case_id = models.CharField(max_length=126)
-
-    class Meta:
-        db_table = "commcare_attendee"
-        indexes = (models.Index(fields=("domain",)),)
+class AttendeeObjectManager(models.Manager):
```

Comment:
I like what you've done here. I'm still uncertain about using a Django model to manage `CommCareCase`s though. I'll play around with this a bit.